plugins/chrony/chrony_drift

```diff
@@ -0,0 +1,92 @@
+#! /bin/sh
+# -*- sh -*-
+
+: <<=cut
+
+=head1 NAME
+
+chrony_drift - Munin plugin to monitor the Chrony drift value.
+
+=head1 APPLICABLE SYSTEMS
+
+Any chronyd host.
+
+=head1 CONFIGURATION
+
+The following configuration parameters are used by this plugin:
+
+ [chrony_drift]
+ env.driftfile - Path to driftfile.
+
+=head2 DEFAULT CONFIGURATION
+
+ [chrony_drift]
+ env.driftfile "/var/lib/chrony/chrony.drift"
+
+=head1 USAGE
+
+Link this plugin to /etc/munin/plugins/ and restart the munin-node.
+
+=head1 AUTHOR
+
+HORINOUCHI Masato <thermes+github@confei.to> 2019-05-26
+
+=head1 LICENSE
+
+Same as munin.
+
+=head1 MAGIC MARKERS
+
+#%# family=auto
+#%# capabilities=autoconf
+
+=cut
+
+driftfile=${driftfile:-'/var/lib/chrony/chrony.drift'}
+
+do_autoconf () {
+    if [ -r $driftfile ]; then
+        echo "yes"
+    else
+        echo "no (could not read driftfile '$driftfile'.)"
+    fi
+}
+
+
+do_config () {
+    cat <<'EOM'
+graph_title Chrony Drift
+graph_args --base 1000
+graph_vlabel Parts Per Million
+graph_category time
+drift.label System Clock Gain/Loss Rate
+error_bound.label Estimate of Error Bound
+graph_info The rate at which the system clock gains or loses time relative to real time.
+EOM
+}
+
+
+do_ () {
+    if [ -r $driftfile ]; then
+        echo "drift.value $(cat $driftfile | awk '{print $1;}')"
+        echo "error_bound.value $(cat $driftfile | awk '{print $2;}')"
+    else
+        echo "Could not read driftfile '$driftfile'." >&2
```

Comment:
In this case you should not error out, but respond with the value `U` (i.e. "unknown").